OpenTelemetry.sln

```diff
@@ -137,11 +137,11 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "TestApp.AspNetCore.2.1", "t
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "OpenTelemetry.Exporter.ZPages.Tests", "test\OpenTelemetry.Exporter.ZPages.Tests\OpenTelemetry.Exporter.ZPages.Tests.csproj", "{98F9556B-116F-49B5-9211-BB1D418446FF}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "OpenTelemetry.Samples.AspNet", "samples\AspNet\OpenTelemetry.Samples.AspNet.csproj", "{9A4E3A68-904B-4835-A3C8-F664B73098DB}"
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "OpenTelemetry.Examples.AspNet", "examples\AspNet\OpenTelemetry.Examples.AspNet.csproj", "{9A4E3A68-904B-4835-A3C8-F664B73098DB}"
 EndProject
-Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "OpenTelemetry.Samples.Console", "samples\Console\OpenTelemetry.Samples.Console.csproj", "{FF3E6E08-E8E4-4523-B526-847CD989279F}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "OpenTelemetry.Examples.Console", "examples\Console\OpenTelemetry.Examples.Console.csproj", "{FF3E6E08-E8E4-4523-B526-847CD989279F}"
 EndProject
-Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "OpenTelemetry.Samples.AspNetCore", "samples\AspNetCore\OpenTelemetry.Samples.AspNetCore.csproj", "{0935622B-9377-4056-8343-AE6ECDC274CF}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "OpenTelemetry.Examples.AspNetCore", "examples\AspNetCore\OpenTelemetry.Examples.AspNetCore.csproj", "{0935622B-9377-4056-8343-AE6ECDC274CF}"
```

Comment:
A general question - should we keep `OpenTelemetry` namespace in the examples or not?
I suggest we remove it for the following reasons:
1. It will be weird for the real users to put `OpenTelemetry` as their namespace, the examples don't seem to lead the right direction.
2. Having the examples under `OpenTelemetry` namespace implies that things under `OpenTelemetry` top level namespace are imported implicitly.

My argument would be that a hello world app using `System.Console.WriteLine` doesn't have to be `System.Examples.HelloWorld`.

Reply:
That's a good idea. i will update this PR to update from OT.Examples.** to Examples.**

Reply:
Or something like this https://github.com/dotnet/samples/blob/master/csharp/list-quickstart/Program.cs.

Reply:
About the `System.Console.Writeline`, since our namespace is `xxx.Console`, we will have to keep using `System`.